base/stats.go

```diff
@@ -461,6 +461,8 @@ type DatabaseStats struct {
 	DocWritesXattrBytes *SgwIntStat `json:"doc_writes_xattr_bytes"`
 	// Highest sequence number seen on the caching DCP feed.
 	HighSeqFeed *SgwIntStat `json:"high_seq_feed"`
+	// Number of bytes written over public interface for REST api
+	HTTPBytesWritten *SgwIntStat `json:"http_bytes_written"`
```

Comment:
Where are we accounting for request body sizes? (here seems like response body size only)

Reply:
Split to https://issues.couchbase.com/browse/CBG-3163